lib/tty/prompt/multi_list.rb

```diff
@@ -19,6 +25,7 @@ class MultiList < List
       def initialize(prompt, **options)
         super
         @selected = SelectedChoices.new
+        @select_keys = init_select_keys(options.fetch(:select_keys, DEFAULT_SELECT_KEYS))
```

Comment:
Let's delay evaluation and move it to a block similar to `:confirm_keys`:

```ruby
@select_keys = init_select_keys(options.fetch(:select_keys) do 
                                  DEFAULT_SELECT_KEYS
                                end)
```

Reply:
I'm also changing this to `self.class::DEFAULT_SELECT_KEYS`, in case someone wants to sub-class and override it.